themes-default/slim/src/components/helpers/config-scene-exceptions.vue

```diff
@@ -0,0 +1,356 @@
+<template>
+    <div class="select-list max-width">
+        <ul>
+            <li v-for="exception of items" :key="`${exception.title}-${exception.season}`">
+                <div class="input-group form-inline" :disabled="!exception.custom">
+                    <input class="form-control input-sm" type="text" :value="exception.title" :disabled="!exception.custom">
+
+                    <select
+                        name="scene-exception-season"
+                        class="select-season"
+                        v-model="exception.season"
+                        :disabled="!exception.custom"
+                    >
+                        <option v-for="season in availableSeasons" :value="season.value" :key="season.value">
+                            {{ season.description }}
+                        </option>
+                    </select>
+
+                    <div
+                        v-if="!exception.custom"
+                        class="external-scene-exception"
+                        v-tooltip.right="'This exception has been automatically added through an automated process sourcing title aliases from medusa.github.io, thexem.de or anidb.info'"
```

Comment:
Instead of `medusa.github.io` I'd just say `Medusa's own exceptions` or similar. 
But other than that, good idea.

Reply:
But `medusa own exceptions` also yields confusion. Because medusa is the app the user installed.

But I understand. I'll think about an alternative

Reply:
Hope its better now